### PR TITLE
chore: add tag-epic to license headers

### PR DESCRIPTION
**Pull request by MaskDuck**
## Summary

add tag-epic to license headers

## Checklist

- [x] If code changes were made then they have been tested.
    - [x] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Review by koala9712 — APPROVED** [review]
Maybe do all of them in one PR?

**Comment by koala9712** [comment]
I don't see any code changes, yet it was checked in the PR description checklist?
Same for "adds something new", this does not add anything new.

**Comment by koala9712** [comment]
This also may cause some merge conflicts, #608 for example
Please keep in mind the years you put.

File: nextcord/raw_models.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 """
 The MIT License (MIT)
 
-Copyright (c) 2015-present Rapptz
+Copyright (c) 2015-2021 Rapptz
+Copyright (c) 2021-present tag-epic
```
Comment (Skelmis) [inline nextcord/raw_models.py:5]: This date is inconsistent with other files in this pr

**Review by Skelmis — CHANGES_REQUESTED** [review]

**Comment by ooliver1** [comment]
its coming up to 2 months, any update on this @MaskDuck?

**Comment by MaskDuck** [comment]
i think you can merge it now

File: nextcord/raw_models.py (hunk 1 above)
Comment (ooliver1) [reply nextcord/raw_models.py:5]: @MaskDuck you marked this as resolved without
- discussion
- commits

Comment (MaskDuck) [reply nextcord/raw_models.py:5]: the file is edited in 2021

**Review by Skelmis — APPROVED** [review]

**Review by ooliver1 — APPROVED** [review]

**Pull request merged**